openreview/invitations/templates.py

```diff
@@ -167,6 +167,54 @@ def __init__(self, name, conference_id, duedate = 0,
     def add_process(self, process):
         self.process = process.render()
 
+class Recruitment(openreview.Invitation):
+    def __init__(self, id, conference_id, duedate = 0,
+        process = None, web = None, inv_params = {}, reply_params = {}, content_params = {}):
+
+        self.id = id
+        self.conference_id = conference_id
+
+        default_inv_params = {
+            'id': self.id,
+            'readers': ['everyone'],
+            'writers': [self.conference_id],
+            'invitees': ['~'],
+            'signatures': [self.conference_id],
+            'duedate': duedate,
+            'process': process,
+            'web': web
+        }
+
+        default_reply_params = {
+            'forum': None,
+            'replyto': None,
+            'readers': {
+                'values': ['~Super_User1']
+            },
+            'signatures': {
+                'values-regex': '\\(anonymous\\)'
+            },
+            'writers': {
+                'values-regex': '\\(anonymous\\)'
+            }
+        }
+
```

Comment:
Why use anonymous? Only a Super User can see them.

Reply:
this is so that they can sign the note without being logged in. any value other than anonymous requires a user token